workers/unity/Assets/Playground/Config/BuildConfiguration.asset

```diff
@@ -30,7 +30,7 @@ MonoBehaviour:
     CloudBuildConfig:
       BuildTargets:
       - Options: 0
-        Enabled: 0
+        Enabled: 1
```

Comment:
QQ when you build a windows x86 client is it still zipped into `WorkerType@Windows.zip` ? Or will be it something like `WorkerType@Windows86.zip` ?

Reply:
x86 and x64 clients share the same zip name `WorkerType@Windows.zip`

need to think about how to best ensure we test the x64 clients in release-qa

Reply:
it now will be `WorkerType@Windows86.zip`